intake_geopandas/geopandas.py

```diff
@@ -1,52 +1,30 @@
 # -*- coding: utf-8 -*-
-from . import __version__
-from intake.source.base import DataSource, Schema
+from abc import ABC, abstractmethod
 
-import json
-import dask.dataframe as dd
-from datetime import datetime, timedelta
+from intake.source.base import DataSource, Schema
+import geopandas
 
+from . import __version__
 
```

Comment:
Should be 2 blank lines?

Reply:
Seems to happen throughout between classes.

Reply:
Sure thing. For future reference, do you have a preferred linter/formatter for this project?

Reply:
flake8 is typical in many of the projects I am involved in. This project is young enough that I wouldn't bother enforcing it, though.